### ISSUE TITLE
Ops in some ONNX models

### ISSUE THREAD

**Issue by tungld** [onnx/onnx-mlir#19]
Here, we enumerate ops used in some popular models, and summarize their implementing status in onnx-mlir. An op is available once its backend tests have passed.

(PR: means still in pull request, Y: Yes)

| Operator           | Available in onnx-mlir | [MNIST (opset v8)]( https://github.com/onnx/models/tree/master/vision/classification/mnist) | [ResNet50 (opset v8)]( https://github.com/onnx/models/tree/master/vision/classification/resnet/resnet50) |
|--------------------|-----------|-------|--------------|
| Add                | Y         | Y     |              |
| AveragePool        |    Y       |       | Y `(No pads; strides: (1, 1))`           |
| BatchNormalizationTestMode |   Y        |       | Y           |
| ConvNoBias               |   Y        | Y `(No bias; pads: (2, 2, 2, 2); strides: (1, 1)`    | Y `(No bias; pads: (0, 0, 0, 0), (1, 1, 1, 1) and (3, 3, 3, 3); strides: (1, 1) and (2, 2))`           |
| Gemm               | Y     |       | Y            |
| MaxPoolSingleOut            |   Y        | Y `(No pads; strides: (2, 2))`     | Y  `(Pads: (1, 1, 1, 1); strides: (2, 2))`          |
| MatMul             |    Y       | Y     |              |
| Relu               | Y         | Y     | Y            |
| Reshape            | Y   | Y     | Y            |
| Softmax            | Y         |       | Y            |
| Sum                | Y         |       | Y            |
|                    |           |       |              |
